UP_onboarding_system/point_of_sale/serializers.py

```diff
@@ -0,0 +1,122 @@
+from dataclasses import field
+from .models import *
+from rest_framework import serializers
+from django.contrib.auth import authenticate, login
+from rest_framework_simplejwt.tokens import RefreshToken
+
+
+class ProfileSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Profile
+        fields = ("name", "role",)
+
+
+class UserRegistrationSerializer(serializers.ModelSerializer):
+    profile = ProfileSerializer()
+    password = serializers.CharField(style={'input_type': 'password'}, write_only=True)
```

Comment:
Can take password2  and  reassert that is  password1

Reply:
Done